multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -50,6 +50,7 @@
 SCORE = "score"
 
 # Metric
+MAP = "map"
```

Comment:
Not sure which one is better but we can consider use “mAP”.

Reply:
Yes, mAP is more accurate.